docs/source/quick-start.ipynb

```diff
@@ -604,7 +604,7 @@
      "name": "stderr",
```

Comment:
<p>Can probably import <code>matplotlib</code>  at the top of the notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xarray-contrib/xskillscore/pull/225/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='68'/>